docs/getting-started.md

```diff
@@ -29,3 +29,79 @@ The author of this project strongly believes descriptors will be a big part of t
 technology and tooling of Bitcoin evolves and changes (Schnorr signatures, Taproot, etc).
 
 To learn more, check out the specific [Descriptors section](/descriptors).
+
+The following code shows how to generate a random mnemonic, the extended (and deterministic) keys from that mnemonic and finally the descriptors from the extended private keys.
+
+To be able to run this code, the `bdk` dependency in `Cargo.toml` must be set as follows:
+
+```
+[dependencies]
+bdk = { version = "0.15.0", default-feature = false, features = ["compiler", "all-keys"] }
+```
+
+```rust
+use bdk::bitcoin::secp256k1::Secp256k1;
+use bdk::bitcoin::util::bip32::{DerivationPath, KeySource};
+use bdk::bitcoin::Network;
+use bdk::descriptor::Segwitv0;
+use bdk::keys::{GeneratableKey, DescriptorKey, ExtendedKey};
+use bdk::{keys::GeneratedKey};
+use bdk::keys::bip39::{Mnemonic, WordCount, Language};
+use bdk::miniscript;
+use bdk::keys::{DerivableKey, DescriptorKey::Secret};
+
+use std::str::FromStr;
+
+fn get_descriptors(network: Network, mnemonic_words: &String) -> (String, String) {
+
+    // Parse a mnemonic
+    let mnemonic  = Mnemonic::parse(mnemonic_words).unwrap();
+
+    // Generate the extended key
+    let xkey: ExtendedKey = mnemonic.into_extended_key().unwrap();
+
+    // Get xprv from the extended key
+    let xprv = xkey.into_xprv(network).unwrap();
+
+    // Create derived privkey from the above xpriv (master key)
+    // The following dereivation paths will be used for receive and change keys
+    // receive: "m/84h/1h/0h/0"
+    // change: "m/84h/1h/0h/1"
+    let mut keys= Vec::new();
+
+    // Create a new Secp256k1 context
+    let secp = Secp256k1::new();
+
+    for path in ["m/84h/1h/0h/0", "m/84h/1h/0h/1"] {
+        let derived_path = DerivationPath::from_str(path).unwrap();
+        let derived_xpriv = &xprv.derive_priv(&secp, &derived_path).unwrap();
+        let origin: KeySource = (xprv.fingerprint(&secp), derived_path);
+        let derived_xprv_descriptor_key: DescriptorKey<Segwitv0> =
+            derived_xpriv.into_descriptor_key(Some(origin), DerivationPath::default()).unwrap();
+
+
+        // Wrap the derived key with the "wpkh()" string to produce a descriptor text
+        if let Secret(key, _, _) = derived_xprv_descriptor_key {
```

Comment:
A bit cleaner way to do this is to use the bip84 template, details here: 
https://docs.rs/bdk/0.15.0/bdk/descriptor/template/struct.Bip84.html

Reply:
I think the example in Bip84 template is useful to show how we can use `bitcoin::utill` to do the same thing. Where as here we wanna show how to use bdk api to do it. I was looking for the same and ended up doing it this way in my rpc tutorial, to only use things exposed in bdk.

I am still not fully satisfied with the amount of code it needs to create descriptors. And probably there's room to compact this out further.

Reply:
I've also have another version with the code below:

```rust
pub fn mnemonic_to_xprv(network: &Network, mnemonic_words: &str) -> ExtendedPrivKey {
    let mnemonic  = Mnemonic::parse(mnemonic_words).unwrap();
    let xkey: ExtendedKey = mnemonic.into_extended_key().unwrap();
    let xprv = xkey.into_xprv(*network).unwrap();
    xprv
}

let wallet_name = wallet_name_from_descriptor(
    Bip84(xpriv.clone(), KeychainKind::External),
    Some(Bip84(*xpriv, KeychainKind::Internal)),
    *network,
    &Secp256k1::new()
 ).unwrap();
    
println!("mnemonic: {}\n\nrecv desc (pub key): {:#?}\n\nchng desc (pub key): {:#?}",
    mnemonic_words,
    wallet.get_descriptor_for_keychain(KeychainKind::External).to_string(),
    wallet.get_descriptor_for_keychain(KeychainKind::Internal).to_string());
 ```
 
I can change this PR to this version, but this one requires a wallet instance to be created before the descriptors are extracted.

Reply:
Ah I see. I think I missunderstood @notmandatory 's comment. Yes it makes sense to use `Bip84` template derivation instead of manually wrapping in `wpkh` string. 

This will be a good addition to have..

Reply:
I updated the PR with this code in 60b72fe